src/javascript/app/Stores/Modules/Trading/trade-store.js

```diff
@@ -384,6 +384,17 @@ export default class TradeStore extends BaseStore {
             has_only_forward_starting_contracts =
                 ContractType.getContractCategories().has_only_forward_starting_contracts;
         }
+
+        if (Object.keys(obj_new_values).includes('duration_unit')) {
```

Comment:
Can this be moved to componentDidMount in duration-wrapper.jsx? also I think we want to keep the duration units separate between simple/advanced